db/migrate/20200625081552_add_unique_index_to_lookup_value.rb

```diff
@@ -1,5 +1,13 @@
 class AddUniqueIndexToLookupValue < ActiveRecord::Migration[6.0]
-  def change
+  def up
+    LookupValue.where.not(id: LookupValue.group(:lookup_key_id, :match).pluck('MAX(id)')).each do |lookup_value|
```

Comment:
I just realized this actually triggers two queries, which can be a problem if you have many thousands of values. The same should work with a nested query:
```suggestion
    LookupValue.where.not(id: LookupValue.group(:lookup_key_id, :match).select('MAX(id)')).each do |lookup_value|
```